packages/card/components/Card.tsx

```diff
@@ -14,6 +14,13 @@ export interface CardProps extends React.HTMLProps<HTMLDivElement> {
    */
   paddingSize?: SpaceSize;
   children?: React.ReactNode | string;
+  /**
+   * The header will be set to the top section of the card optionally. Set a header image along with corresponding alt text.
+   */
+  header?: {
+    headerImg?: string;
```

Comment:
❓ - if the header is optional, should these be mandatory? Thinking out loud more than anything.